FRIEND/FRIEND.cpp

```diff
@@ -663,6 +667,8 @@ class FRIEND final : public PluginDelegate
 				}
 				
 				// try to generate register/instruction hint otherwise
+				if (el->op != cot_str)
```

Comment:
OK, please revert this change. The problem is not that trivial as it looks, `op` is not necessarily `cot_str` it is not even `cot_helper` sometimes (it depends on decompiler). I am aware that it causes crash and working on a solution. I will commit my fix right after your merge.